lib/seattleflu/db/cli/command/etl/presence_absence.py

```diff
@@ -0,0 +1,386 @@
+"""
+Process enrollment documents into the relational warehouse
```

Comment:
There's a good bit of useful info the commit message that I think could be adapted to be part of this command's `--help`. For example, see `id3c identifier --help` and https://github.com/seattleflu/id3c/blob/3be10a059c526610f657a85b891c8cb281d67ed2/lib/seattleflu/db/cli/command/identifier.py#L1-L22.

Reply:
We added a good chunk of the commit we think you're referencing to the top of the `presence_absence.py` file. We're happy to talk about this in person on Monday